module/Finna/src/Finna/Content/Covers/Kirjavalitys.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/**
+ * Kirjavalitys Cover Image Service cover content loader.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) The National Library of Finland 2018.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Content
+ * @author   Kalle Pyykkönen <kalle.pyykkonen@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+
+namespace Finna\Content\Covers;
+
+use VuFindCode\ISBN;
+
+/**
+ * Kirjavalitys Cover Image Service cover content loader.
+ *
+ * @category VuFind
+ * @package  Content
+ * @author   Kalle Pyykkönen <kalle.pyykkonen@helsinki.fi>
```

Comment:
This is a name I haven't heard in a long time.

Reply:
Yes, but since this is partly copied from BTJ, he's the original author. I added myself too now. :)